_oasis

```diff
@@ -56,7 +56,7 @@ Library xenopsd
     Xenops_server_plugin,
     Xenops_server_skeleton,
     Scheduler,
-    Path,
+    Xpath,
```

Comment:
I can't think of anything else that may be using it. I think that the lib is just used by the multiple backends inside xenopsd.git.